third_party/move/move-prover/boogie-backend/src/prelude/prelude.bpl

```diff
@@ -3,7 +3,7 @@ Copyright (c) The Diem Core Contributors
 SPDX-License-Identifier: Apache-2.0
 
 This files contains a Tera Rust template for the prover's Boogie prelude.
-(See https://tera.netlify.app/docs).
+(See https://docs.rs/tera/latest/tera/).
```

Comment:
not sure if this is a relevant change?

Reply:
Slipped in by accident but also doesn't hurt (the old link is dead).